server_cmds_test.go

```diff
@@ -306,19 +306,98 @@ func testCmdsServer(t *testing.T, c *Client) {
 
 		expected := []*OnlineClient{
 			{
-				ID:          5,
-				ChannelID:   7,
-				DatabaseID:  40,
-				Nickname:    "ScP",
-				Type:        0,
-				Away:        true,
-				AwayMessage: "not here",
+				ID:                             42087,
+				ChannelID:                      39,
+				DatabaseID:                     19,
+				Nickname:                       "bdeb1337",
+				Type:                           0,
+				Away:                           nil,
+				AwayMessage:                    nil,
+				FlagTalking:                    nil,
+				InputMuted:                     nil,
+				OutputMuted:                    nil,
+				InputHardware:                  nil,
+				OutputHardware:                 nil,
+				TalkPower:                      nil,
+				IsTalker:                       nil,
+				IsPrioritySpeaker:              nil,
+				IsRecording:                    nil,
+				IsChannelCommander:             nil,
+				UniqueIdentifier:               nil,
+				ChannelGroupID:                 nil,
+				ChannelGroupInheritedChannelID: nil,
+				Version:                        nil,
+				Platform:                       nil,
+				IdleTime:                       nil,
+				Created:                        nil,
+				LastConnected:                  nil,
+				IconID:                         nil,
+				Country:                        nil,
+				IP:                             nil,
+				Badges:                         nil,
+				ServerGroups:                   nil,
```

Comment:
Since nil is the empty value no need to define.
```suggestion
```

Reply:
Thank you. Understood & edited in the new commit.